cmds/webboot/webboot_test.go

```diff
@@ -161,3 +161,33 @@ func TestGetCachedDirectory(t *testing.T) {
 		}
 	}
 }
+func TestBackOption(t *testing.T) {
+	uiEvents := make(chan ui.Event)
+	input := []string{"1", "<Enter>", "1", "<Enter>", "0", "<Enter>", "1", "<Enter>", "0", "<Enter>", "0", "<Enter>"}
+	go pressKey(uiEvents, input)
+
+	var entry menu.Entry = &DirOption{path: "./testdata"}
+	var currentPath string = ""
+	var err error = nil
+	for i := 0; i < 9; i++ {
```

Comment:
Hm I only looked at this briefly but why do we need this loop for 9 iterations? Is it possible to simplify this test to only 1) call exec for DirOption(/testdata), 2) choose 1 thing, 3) go back and confirm that the new option after going back has the correct path?

Reply:
I just think it’s easier to detect problems with a series of complicated instructions. But I agree with you now. Anyway, just keep it simple